fuse/dir.go

```diff
@@ -66,7 +67,9 @@ func (d *dir) Lookup(ctx context.Context, req *fuse.LookupRequest, resp *fuse.Lo
 	}
 
 	log.Debugf("FUSE: Found file %v/%v", d, cname)
-	return newFile(d, entry), nil
+	f := newFile(d, entry)
+	f.fillAttr(&resp.Attr)
```

Comment:
What's the benefit of doing this here?

Reply:
May avoid a separate call to Attr, but it's probably not necessary.